workspaces/bulk-import/plugins/bulk-import/src/components/PreviewFile/PreviewPullRequest.test.tsx

```diff
@@ -262,6 +278,8 @@ describe('Preview Pull Request', () => {
     const { getByTestId, getByText } = render(
       <TestApiProvider
         apis={[
+          [errorApiRef, new MockErrorApi()],
+          [translationApiRef, mockApis.translation()],
```

Comment:
May be you could create `renderWithProviders` for this as well , it might reduce the code duplication to below 3